plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/preferences/OpenShiftPreferencePage.java

```diff
@@ -63,8 +66,17 @@ public void createFieldEditors() {
 		this.cliLocationEditor.setFilterPath(SystemUtils.getUserHome());
 		this.cliLocationEditor.setFileExtensions(EXTENSIONS);
 		addField(cliLocationEditor);
-		
-	}
+		//TODO Fix the Layout
+        Link link = new Link(getFieldEditorParent(), SWT.WRAP );
+        link.setText("Openshift plugin needs the openshift client binary (CLI) to provide some features like port forwarding.<br>"
```

Comment:
The OpenShift Client binary (oc) is required for features such as Port Forwarding or Log Streaming.